packages/vite/src/node/server/environment.ts

```diff
@@ -32,6 +31,19 @@ import {
 import type { RemoteEnvironmentTransport } from './environmentTransport'
 import { isWebSocketServer } from './ws'
 
+export function getDefaultResolvedEnvironmentOptions(
+  config: ResolvedConfig,
+): ResolvedEnvironmentOptions {
+  return {
+    define: config.define,
+    resolve: config.resolve,
+    consumer: 'server',
+    webCompatible: false,
+    dev: config.dev,
+    build: config.build,
+  }
+}
+
```

Comment:
I think it is better to move this one level up to `/node/baseEnvironment` so we don't need to include `/server/` from `build.ts`